hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/store/UpdatedAccountTracker.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.evm.store;
+
+import org.hyperledger.besu.datatypes.Address;
+
+public interface UpdatedAccountTracker {
```

Comment:
Perhaps we could add some javadoc. A first draft might be,
```
/**
* When an EVM call changes an account's balance, this change needs to be immediately visible
* to any Hedera system contract that runs within the same frame. (I.e., we cannot wait for commit()
* to be called on a WorldUpdater; we need to immediately update the "parallel" Hedera world state).
* 
* <p>This type defines the callback a mutated account uses to report its balance changes to 
* the Hedera world state when running an EVM transaction in a consensus node. (Accounts
* "mutated" during an eth_estimateGas call running on a mirror node don't actually change, so
* we can provide a null implementation of the callback there.)
* */
```

Reply:
Added.